cargo-near/src/util/tracking.rs

```diff
@@ -0,0 +1,47 @@
+use chrono::prelude::*;
+use reqwest::Client;
+use rustc_version_runtime::version;
+use std::env;
+
+#[derive(Debug, serde::Serialize)]
+struct MixpanelProperties {
+  token: String,
+  rustc_version: String,
+  os: String,
+  timestamp: String,
+}
+
+#[derive(Debug, serde::Serialize)]
+struct TrackingData {
+    event: String,
+    properties: MixpanelProperties,
+}
+
+pub(crate) async fn track_usage() {
+    let rustc_version = version();
+    let properties = MixpanelProperties {
+      token: "24177ef1ec09ffea5cb6f68909c66a61".to_string(),
+      rustc_version: format!(
```

Comment:
For cargo near new we will want to fix the value `contract: "rs".to_string()`

If needed by the `cargo near` team we can track the rust version, if not, we can remove it

Reply:
@frol do we need to track the rust version?

Reply:
@garikbesson I'd track CLI version instead of Rust version.